modules/claims_api/spec/support/swagger_shared_components/v2.rb

```diff
@@ -220,6 +233,25 @@ def self.body_examples # rubocop:disable Metrics/MethodLength
             example: disability_compensation_request_body_example
           }
         },
+        sync_disability_compensation: {
+          in: :body,
+          name: 'data',
+          required: true,
+          schema: {
+            type: :object,
+            required: ['data'],
+            properties: {
+              data: {
+                type: :object,
+                required: ['attributes', disability_compensation_synchronous_json_schema['required']],
+                properties: {
+                  attributes: disability_compensation_synchronous_json_schema
+                }
+              }
+            },
+            example: disability_compensation_request_body_example
```

Comment:
Since we do not send in the claimId with the initial request I assumed this could stay the same